src/templates/reusable/project_info_inputs.mdx

```diff
@@ -0,0 +1,18 @@
+
+    The data for this tab should be know prior to the initial site visit.
```

Comment:
See my comment above about leaving this first sentence out of ProjectInfoInputs template.

Reply:
Done